Day-7_Functions/functions.js

```diff
@@ -228,11 +228,20 @@ console.log(reverseArray([1, 2, 3, 4, 5]))
 console.log(reverseArray(['A', 'B', 'C']))
 
 // Declare a function name capitalizeArray. It takes array as a parameter and it returns the - capitalizedarray.
+// Using .forEach()
 function capitalizeArray(arr){
-    const capitalizedArr = []
-    for (let i = 0; i < arr.length; i++) {
-        capitalizedArr.push(arr[i].toString().toUpperCase())
-    }
+    const capitalizedArr = []    
+    arr.forEach(element => {
+        capitalizedArr.push(element.toString().toUpperCase())
+    })
+    return capitalizedArr
```

Comment:
I was hoping you would use the `.map()` method. But your solution is acceptable, too.